source/include/core_mqtt.h

```diff
@@ -67,10 +67,10 @@ struct MQTTDeserializedInfo;
 
 /**
  * @ingroup mqtt_callback_types
- * @brief Application provided function to query the current time in
- * milliseconds.
+ * @brief Application provided function to query the time elapsed since a given
```

Comment:
Could be helpful to specify that this is a monotonic timer rather than related to the current date/time.

Reply:
Done